Source/Core/DolphinQt/Config/Mapping/MappingIndicator.h

```diff
@@ -17,20 +18,37 @@ class ReshapableInput;
 
 class QPaintEvent;
 class QTimer;
+class QPainter;
 
 class MappingIndicator : public QWidget
 {
 public:
   explicit MappingIndicator(ControllerEmu::ControlGroup* group);
 
+  bool is_calibrating = {};
```

Comment:
This should probably be private, like the other member variables (even if just a plain get/set, or in this case `BeginCalibration()` and `EndCalibration()`, I suppose, since it's not actually queried externally, only set)

Reply:
:+1: